### PR TITLE
fix(third_party): fix attribute fields and use id instead

### PR DESCRIPTION
**Pull request by dandhlee**
For UniversalReferences, I added the attribute name in the `type` section by accident, and used `name` for attribute names instead of using `id`. Updating to use appropriate columns instead.

[First commit](https://github.com/googleapis/doc-templates/pull/352/commits/b12f8d78f8b197c07e85cf9d8b222871c253f461) swaps the attribute fields to use id. Also adding `class parametername` to stay consistent with the ManagedReferences docs, although I can revert this if this is not too necessary. Confirmed that there's no more occurrences of `{{{name}}}` unless appropriate.

[Second commit](https://github.com/googleapis/doc-templates/pull/352/commits/689f2d681c0cf6960727ef68daebf39b0d4dada5) updates the Python testdata. Pulled in the latest refresh for more accurate content.

[Third commit](https://github.com/googleapis/doc-templates/pull/352/commits/da87eb382fd0d49bf8efa419281be6a9770bb5e8) updates the goldens based on the new test data and the fix.

Fixes b/218725782.

- [x] Tests pass

### CODE REVIEW

**Comment by dandhlee** [comment]
Test seems to be failing on node, investigating..

**Comment by dandhlee** [comment]
Forgot some Node files 😓

File: third_party/docfx/templates/devsite/partials/uref/class.header.tmpl.partial [hunk 1]
```diff
@@ -18,8 +18,8 @@
 {{/attributes.0}}
 {{#attributes}}
     <tr>
-      <td>{{{var_type}}}</td>
-      <td><code>{{{name}}}</code><br>{{{description}}}</td>
+      <td><span class="parametername">{{{id}}}</span></td>
```
Comment (tbpg) [inline third_party/docfx/templates/devsite/partials/uref/class.header.tmpl.partial]: What is this `span` for? Do we use `class="parametername"` for styling? Seems unrelated?

If anything, I'd prefer to delete this in the ManagedReference template (in a different PR) than add it here.

**Review by tbpg — CHANGES_REQUESTED** [review]

File: third_party/docfx/templates/devsite/partials/uref/class.header.tmpl.partial (hunk 1 above)
Comment (dandhlee) [reply third_party/docfx/templates/devsite/partials/uref/class.header.tmpl.partial]: Sounds good. I'll create an issue to clean those up!

**Comment by dandhlee** [comment]
Filed #353, updated UniversalRef docs to not use the unnecessary span. Please take a look again!

**Review by tbpg — COMMENTED** [review]
Are there any concerns with rollout? What happens before/after the plugin change combined with before/after this change?

**Comment by dandhlee** [comment]
Yeah this is going to be weird to merge this PR, followed by the new release on sphinx plugin, and then another template update (#346). Will merge this content over to #346 instead.
